gems/sorbet/test/snapshot/partial/project-without-tapioca-but-rbis/src/test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+bundle exec "$1" tc
```

Comment:
This test is explicitly not calling `srb init` because that's not the workflow with `tapioca`, right?

Reply:
Exactly. We assume the user will do `tapioca init` instead.